setup.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
 from setuptools import setup, find_packages
-from pkg_resources import require, DistributionNotFound
-import pages
```

Comment:
I seemed to have worked until this point. Django seems to do it as well ? https://github.com/django/django/blob/master/setup.py#L31